mne/viz/_brain/_brain.py

```diff
@@ -2328,7 +2333,6 @@ def add_head(self, dense=True, color=None, alpha=0.5):
             (``head``).
         color : matplotlib-style color | None
```

Comment:
We document everywhere else as just `color`, and no need for `None` here
```suggestion
        color : color
```